setup.py

```diff
@@ -65,6 +65,7 @@
     url='https://github.com/ethereum/web3.py',
     include_package_data=True,
     install_requires=[
+        "content-hash>=1.0.0,<2.0.0",
```

Comment:
This needs some discussion.  Just leaving comment here so it doesn't get merged before resolving this.

Reply:
@pipermerriam I understand concerns of including additional third-party package into Web3.py. However, I can update the requirement to lock `content-hash` to a specific version (currently `1.1.0`) so you would be able to manually review it before updating.

I think that content hash support for ENS should be as important as current ENS functionality. The content hash will really expend possibilities of ENS as ENS domains are not anymore bound only to ETH addresses.